pkg/kube/controllers/extendedjob/job_reconciler.go

```diff
@@ -19,19 +19,25 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
+	bdm "code.cloudfoundry.org/cf-operator/pkg/bosh/manifest"
+	bdv1 "code.cloudfoundry.org/cf-operator/pkg/kube/apis/boshdeployment/v1alpha1"
 	ejapi "code.cloudfoundry.org/cf-operator/pkg/kube/apis/extendedjob/v1alpha1"
 	"code.cloudfoundry.org/cf-operator/pkg/kube/util/config"
 	"code.cloudfoundry.org/cf-operator/pkg/kube/util/ctxlog"
+	store "code.cloudfoundry.org/cf-operator/pkg/kube/util/store/manifest"
 )
 
 // NewJobReconciler returns a new Reconciler
 func NewJobReconciler(ctx context.Context, config *config.Config, mgr manager.Manager, podLogGetter PodLogGetter) (reconcile.Reconciler, error) {
+	manifestStore := store.NewStore(mgr.GetClient(), bdm.DesiredManifestKeyName)
```

Comment:
This is not ok - the `ExtendedJob` should have no dependencies on "bosh manifest" stuff.